src/devices/wmun.ts

```diff
@@ -0,0 +1,36 @@
+import * as fz from "../converters/fromZigbee";
+import * as exposes from "../lib/exposes";
+import * as reporting from "../lib/reporting";
+const e = exposes.presets;
+const zosung = require("zigbee-herdsman-converters/lib/zosung");
+const fzZosung = zosung.fzZosung;
+const tzZosung = zosung.tzZosung;
+const ez = zosung.presetsZosung;
+import type {DefinitionWithExtend} from "../lib/types";
+
+export const definitions: DefinitionWithExtend[] = [
+    {
+        fingerprint: [{modelID: "TS1201", manufacturerName: "_TZ3290_u9xac5rv"}],
```

Comment:
Please add to `tuya.ts` -> `ZS06` instead (use `tuya.whitelabel` for proper identification)

https://github.com/Koenkk/zigbee-herdsman-converters/blob/cba3082d1a9594b4aa74e44d6dccd0ba5d6e1a8b/src/devices/tuya.ts#L9212